webapp/platform/components/src/generic_modal/generic_modal.tsx

```diff
@@ -73,8 +77,39 @@ export class GenericModal extends React.PureComponent<Props, State> {
         };
     }
 
+    componentDidMount(): void {
+        const {focusableElement} = this.props;
+        if (focusableElement && focusableElement.current) {
+            setTimeout(() => {
```

Comment:
If we do need to do this manually, maybe the RB `Modal` has an `onShown` handler we could use instead of needing a `setTimeout`?